src/source_map_utils.ts

```diff
@@ -30,7 +30,8 @@ export function extractInlineSourceMap(source: string): string {
     previousResult = result;
     result = inlineSourceMapRegex.exec(source);
   } while (result !== null);
-  const base64EncodedMap = previousResult![1];
+  if (!previousResult) return null;
+  const base64EncodedMap = previousResult[1];
```

Comment:
I intentionally re-reverted Lucas' commit so I could try this out in google. No longer necessary.